### PR TITLE
Created swiftgen.yml and Localizable.strings files, autogenerated Ass…

### PR DESCRIPTION
**Pull request by santaarturio**
…ets and Strings files

## Description

Added the ability to localize the application

### Types of changes

What types of changes does your code introduce to the Connect app?

- [x] New feature (non-breaking change which adds functionality)

### Checklist

- [x] I have updated configuration files

### CODE REVIEW

File: TopMovies/View/TopMoviesVC/SeeAllMoviesCollectionViewCell.swift [hunk 1]
```diff
@@ -32,7 +32,12 @@ final class SeeAllMoviesCollectionViewCell: UICollectionViewCell {
     }
   }
   private func setupStyle() {
-    seeMoreImageView.image = UIImage(named: "seeAllMovies") ?? UIImage()
+    if #available(iOS 13.0, *) {
+      seeMoreImageView.image = UIImage(asset: Asset.Images.seeAllButton)?
+        .withTintColor(UIColor(asset: Asset.Colors.seeAllButton) ?? .black)
```
Comment (balavor) [inline TopMovies/View/TopMoviesVC/SeeAllMoviesCollectionViewCell.swift]: It doesn't have to be used like that. It generates properties that guarantee that you have an image/color.

File: TopMovies/View/TopMoviesVC/MovieCollectionViewCell.swift [hunk 2]
```diff
@@ -22,12 +22,12 @@ struct MovieCollectionProps {
...
     guard let movie = movie else { return nil }
-    adultLabelText = "Adult: \(movie.adult ? "Yes" : "No")"
-    ratingLabelText = "Rating: \(movie.rating)"
+    adultLabelText = "\(L10n.App.Home.Movie.adult): \(movie.adult ? L10n.App.Home.Movie.yes : L10n.App.Home.Movie.no)"
+    ratingLabelText = "\(L10n.App.Home.Movie.rating): \(movie.rating)"
     titleLabelText = movie.title
     descriptionLabeltext = movie.description
     posterURL = movie.poster
-    posterPlaceholderImage = UIImage(named: "moviePlaceholder") ?? UIImage()
+    posterPlaceholderImage = UIImage(asset: Asset.Images.moviePlaceholder) ?? UIImage()
```
Comment (balavor) [inline TopMovies/View/TopMoviesVC/MovieCollectionViewCell.swift]: SwiftGen is exactly to get rid of those empty images.

File: TopMovies.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -433,6 +451,25 @@
...
+			inputPaths = (
+			);
+			name = SwiftGen;
+			outputFileListPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "if [ \"${CONFIGURATION}\" = \"Debug\" ]; then\nif which swiftgen >/dev/null; then\n/usr/local/bin/swiftgen config run --config \"$SRCROOT/swiftgen.yml\"\nelse\necho \"warning: SwiftGen not installed\"\nfi\nfi\n";
```
Comment (balavor) [inline TopMovies.xcodeproj/project.pbxproj]: Hmm... Maybe SwiftGen should be run for all configurations, not only for debug?

File: TopMovies/View/TopMoviesVC/SeeAllMoviesCollectionViewCell.swift [hunk 4]
```diff
@@ -32,7 +32,12 @@ final class SeeAllMoviesCollectionViewCell: UICollectionViewCell {
     }
   }
   private func setupStyle() {
-    seeMoreImageView.image = UIImage(named: "seeAllMovies") ?? UIImage()
+    if #available(iOS 13.0, *) {
+      seeMoreImageView.image = Asset.Images.seeAllButton.image
+        .withTintColor(Asset.Colors.seeAllButton.color)
+    } else {
+      seeMoreImageView.image = Asset.Images.seeAllButton.image
+    }
```
Comment (balavor) [inline TopMovies/View/TopMoviesVC/SeeAllMoviesCollectionViewCell.swift]: Should this work right here?
```swift
    if #available(iOS 13.0, *) {
       seeMoreImageView.image = Asset.Images.seeAllButton.image
         .withTintColor(Asset.Colors.seeAllButton.color)
     } else {
       seeMoreImageView.image = Asset.Images.seeAllButton.image.withRenderingMode(.alwaysTemplate)
       seeMoreImageView.image.tintColor = Asset.Colors.seeAllButton.color
     }
```

Comment (santaarturio) [reply TopMovies/View/TopMoviesVC/SeeAllMoviesCollectionViewCell.swift]: The .image doesn't have a tintColor property, the ImageView does, but it doesn't work as expected. Actually I should only have 2 image options for any and dark appearance, but if I don't, there is only a way to use the .withTintColor () method if iOS 13 is installed, or use the default image if not.

**Review by balavor — APPROVED** [review]
Now looks safe! :D

**Pull request merged**